docs/index.md

```diff
@@ -5,7 +5,7 @@
   * [1.1 Clinical guidelines for application](#11-clinical-guidelines-for-application)
   * [1.2 Use cases](#12-use-cases)
   * [1.3 Technical specification](#13-technical-specification)
-  * [1.4 Mapping between OIOXML standard (XDIS91), and FHIR CareCommunication](#mapping-between-oioxml-and-fhir)
+  * [1.4 Mapping from OIOXML standard (XDIS91), and FHIR CareCommunication](#mapping-between-oioxml-and-fhir)
```

Comment:
bør der ikke stå Mapping from OIOXML standard (XDIS91) to FHIR CareCommunication?